IHP/FileStorage/ControllerFunctions.hs

```diff
@@ -231,8 +231,8 @@ createTemporaryDownloadUrlFromPathWithExpiredAt validInSeconds objectPath = do
             let url = if any (`isPrefixOf` objectPath) urlSchemes
                     -- BC, before we saved only the relative path of a file, we saved the full URL. So use it as is.
                     then objectPath
-                    -- We have the relative path (prefixed with slash), so add the baseUrl.
-                    else frameworkConfig.baseUrl <> objectPath
+                    -- We have the relative path (without with slash prefix), so add the baseUrl.
+                    else frameworkConfig.baseUrl <> "/" <> objectPath
```

Comment:
Won't this break if existing objectPath's that are already in the database already have a `/` in front?

Reply:
We didn't have a release with that, it was only on `master` for a while

Reply:
But if you think, I can check if `objectPath` already has `/` or not, before crafting the URL

Reply:
I'll adapt PR, to check for the "/", so this function is more robust

Reply:
Improved logic, with test

https://github.com/digitallyinduced/ihp/pull/2059/commits/53baf82a9b2f103760c891380a8d19590a38a33c